### PR TITLE
added new pvforecast adapter 

### PR DESCRIPTION
**Pull request by Patrick-Walther**

### CODE REVIEW

**Comment by Apollon77** [comment]
Please Update the PR to only contain your changes ... it seems that your bracnch was outdated!!

**Review by Patrick-Walther — COMMENTED** [review]
add only my pvforecast information

**Comment by Apollon77** [comment]
RE-CHECK!

**Comment by Patrick-Walther** [comment]
fix the issue

File: sources-dist.json [hunk 1]
```diff
@@ -1567,6 +1567,11 @@
     "icon": "https://raw.githubusercontent.com/ioBroker/ioBroker.pushsafer/master/admin/pushsafer.png",
     "type": "messaging"
   },
+  "pvforecast": {
+    "meta": "https://raw.githubusercontent.com/Patrick-Walther/ioBroker.pvforecast/main/io-package.json",
+    "icon": "https://github.com/Patrick-Walther/ioBroker.pvforecast/raw/main/admin/pvforecast.png",
+    "type": "weather",
```
Comment (ldittmar81) [inline sources-dist.json:1573]: There is an error here. Please delete the last ,

**Review by ldittmar81 — CHANGES_REQUESTED** [review]
There is an error on sources-dist.json

**Comment by Apollon77** [comment]
Hi, here some review feedback:
* first of all as @ldittmar81 already stated ... no github.com links please ... adjust please in repo PR
* Typo in Readme: should be "## License" :-)
* Admin depenendency ned to be in "globalDependencies" usually https://github.com/iobroker-community-adapters/ioBroker.pvforecast/blob/main/io-package.json#L275-L277
* https://github.com/iobroker-community-adapters/ioBroker.pvforecast/blob/main/io-package.json#L387 should be role "json" too (same state below) whth common-type=string
* please remove https://github.com/iobroker-community-adapters/ioBroker.pvforecast/blob/main/main.js#L199-L207 if not needed
* please use ` 'system.adapter.' + this.namespace` here https://github.com/iobroker-community-adapters/ioBroker.pvforecast/blob/main/main.js#L501 for "from"
* please check the weather roles vs https://github.com/ioBroker/ioBroker/blob/master/doc/STATE_ROLES.md#weather ... maybe better matches are possible
Thank you for checking and adjusting
*

**Comment by MeisterTR** [comment]
@Apollon77 all fixed

**Comment by Apollon77** [comment]
I have adjusted the Repo PR

**Comment by Apollon77** [comment]
Verified, then npm release of this version please and we are ready to merge

**Comment by MeisterTR** [comment]
@Apollon77  ready to merge?

**Comment by Apollon77** [comment]
Put 1.0.1 in it for stabe

**Comment by Apollon77** [comment]
Sorry for the delay, welcome in the repo

**Pull request merged**